File: jzoffer41-67/64.cpp
class Solution {
public:
    vector<int> maxInWindows(const vector<int>& num, unsigned int size)
    {
        vector<int>max;
        if(num.empty()||size>num.size()||size<1)
            return max;
        int m;
        for(int i=0;i<num.size()-size+1;i++)
            {
            m=num[i];
            for(int j=i+1;j<i+size;j++)
            {
             
            if(num[j]>m)
                {
                m=num[j];
            } 
         }
            max.push_back(m);
        }
             
       return max;    
    }
};

File: jzoffer41-67/64.js
function maxInWindows(num, size)
{
    // write code here
    let max=[];
    if(size==0||num.length==0||size>num.length) return max;
    for(let i=0;i<num.length-size+1;i++){
        let m=num[i];
        for(let j=i+1;j<i+size;j++){
            if(num[j]>m) m=num[j];
        }
        max.push(m);
    }
    return max;
}